lib/docker-sync/dependencies/fswatch.rb

```diff
@@ -0,0 +1,14 @@
+module DockerSync
+  module Dependencies
+    module Fswatch
+      def self.available?
+        return @available if defined? @available
```

Comment:
I see, no i do not mind it, but you already have that in https://github.com/EugenMayer/docker-sync/blob/47f1c6cf08f583a77355ce942a7a64f4c1841fd1/lib/docker-sync/dependencies/docker_driver.rb#L6 so i was not sure you want to keep it that dry